harness/utils/rate_limiter.py

```
@@ -65,17 +65,17 @@ def acquire(self, token_cost: float, req_cost: float = 1.0) -> None:
-                    pass
-                self.cond.wait(timeout=min(max(wait_req, 0.01), max(wait_tok, 0.01), 5.0))
+                # Only log significant waits (>5s) to reduce spam
+                if wait_s > 5.0:
+                    try:
+                        print(
+                            f"[rate-limit] {self.name}: sleeping ~{wait_s:.1f}s "
+                            f"(need tok={need_tok:.0f}; cap tpm={self.tok_capacity:.0f}/m)",
+                            flush=True
+                        )
+                    except Exception:
+                        pass
```

Ruff (0.14.3)

`try`-`except`-`pass` detected, consider logging the exception

(S110)